### PR TITLE
feat(@angular/cli): update webpack to latest version

### PR DESCRIPTION
**Pull request by hansl**
We dont want this in 1.3.x, so marking as a feature.

### CODE REVIEW

File: tests/e2e/utils/fs.ts [hunk 1]
```diff
@@ -120,8 +120,15 @@ export function replaceInFile(filePath: string, match: RegExp | string, replacem
 
 
 export function appendToFile(filePath: string, text: string, options?: any) {
-  return readFile(filePath)
-    .then((content: string) => writeFile(filePath, content.concat(text), options));
+  return new Promise<void>((resolve, reject) => {
```
Comment (filipesilva) [inline tests/e2e/utils/fs.ts]: Nit: should this be a in separate commit?

Comment (hansl) [inline tests/e2e/utils/fs.ts]: it is?
